- added package json

nitreo opened the pull request.

pgilmorepf commented on PlayFabSdk/package.json:
```diff
@@ -0,0 +1,13 @@
+  "description": "Playfab SDK for JS client applications",
+  "license": "Apache-2.0",
+  "scripts": {
+    "test": ".\\tests\\run.bat"
```
There is no tests folder...

nitreo commented:
OK I removed {script:tests} and version things.
1. To run tests, we have to configure some complex stuff like silenium driver, and make it run the qunit. It is possible and I can take care of it, but you gotta tell me explicitly that you want it, as it might be something complicated

2. I removed version because I don't know how to figure current package version...

pgilmorepf commented:
Moved this commit to the SdkGenerator, and accepted.
https://github.com/PlayFab/SDKGenerator/pull/116
